src/exchange/ExchangeHomeScreen.tsx

```diff
@@ -93,6 +93,9 @@ function ExchangeHomeScreen({ navigation }: Props) {
   const localCurrencyCode = null
   const localExchangeRate = useSelector(getLocalCurrencyToDollarsExchangeRate)
   const currentExchangeRate = useDollarToCeloExchangeRate()
+  const shouldDisplayWithdrawCelo = useTypedSelector(
+    (state) => state.app.celoWithdrawalEnabledInExchange
```

Comment:
could we create a selector for this? then we can avoid using both the `useTypedSelector` and `useSelector` here and keep consistency with the other selectors in this file :)

Reply:
Fixed in cab51ff4ab45c4802e7148642d9c928d06b91a42!

Reply:
oh sorry if i wasn't clear before, but i meant using a selector like `useSelector(celoWithdrawalEnabledInExchangeSelector)` and abstracting the selector internals to the selector file. The benefit of doing this aside from consistency is the component doesn't need to know exactly where in the state this information comes from :)